src/deckgl-layers/scatterplot-brushing-layer/scatterplot-brushing-layer.js

```diff
@@ -54,9 +53,9 @@ export default class ScatterplotBrushingLayer extends ScatterplotLayer {
   initializeState() {
     super.initializeState();
 
-    this.getAttributeManager().addInstanced({
-      instanceFilterValue: {size: MAX_GPU_FILTERS, accessor: 'getFilterValue'}
-    });
+    // this.getAttributeManager().addInstanced({
```

Comment:
do you still need this one? otherwise you can delete it

Reply:
Need it later to implement deck.gl filter attributes